Frontend

gobyrne opened the pull request.

gobyrne committed:
Small improvements for creating text commands

kheaactua committed:
Merge pull request #11 from kheaactua/stage

Removed host specific stuff

gobyrne committed:
Brocken GET request for orders

gobyrne committed:
Semi-working GET to orders route

gobyrne committed:
Merge pull request #13 from kheaactua/frontend

For matt testing Orders route

kheaactua committed:
Changed alerts to console.debug()s

kheaactua committed:
Merge branch 'master' into frontend

gobyrne committed:
more testing

kheaactua committed:
Whitespace

gobyrne committed:
Merge remote-tracking branch 'origin/frontend' into frontend

Conflicts:
	app/app.js

kheaactua committed:
Changed appostrophies to quotations, and got rid of weird self reference to the controller

kheaactua committed:
modeline

kheaactua committed:
Removed this in favour of $scope, and got ng-repeat loops working.  This can't iterate over objects, only arrays

kheaactua committed:
Merge branch 'frontend' of github.com:kheaactua/diplomacy-engine into frontend

Conflicts:
	app/app.js

kheaactua committed:
Typo

gobyrne committed:
New server and removing extra braces

Small commit before I add all the packages

gobyrne committed:
Revert "New server and removing extra braces"

This reverts commit 8fdb4ad7b289f3b86c5bbca03cf0a0b2d86fe296.

gobyrne committed:
Merge pull request #15 from kheaactua/master

Typo

gobyrne committed:
Local packages

jquery, angularjs and bootstrap saved locally and managed with bower

kheaactua commented on app/bower_components/angularjs/.bower.json:
```diff
@@ -0,0 +1,18 @@
+{
```
Please remove this file (everything in `bower_components/`)

kheaactua commented on app/index.html:
```diff
@@ -1,9 +1,9 @@
-	<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
+    <script type="text/javascript" src="/app/bower_components/jquery/dist/jquery.js"></script>
+    <script type="text/javascript" src="/app/bower_components/angularjs\angular.js"></script>
+	<link rel="stylesheet" type="text/css" href="/app/bower_components/bootstrap/dist/css/bootstrap.css" />
```
Might want to use the `.min.js` versions.  Well, up to you.  These are easier to debug, but for production we'd want to change.  (Well, if we were in a proper production mode, we'd have a system that built them automatically, and wouldn't have to think about it.)

gobyrne replied:
Good point, but lets leave it for now... it might actually help for something.

kheaactua commented on app/bower_components/jquery/src/wrap.js:
```diff
@@ -0,0 +1,79 @@
+	"./traversing" // parent, contents
+], function( jQuery ) {
+
+jQuery.fn.extend({
```
Can you comment what this does/what it's for?

gobyrne committed:
Removing all those packages...

Was never supposed to commit them in the first place.

gobyrne committed:
Ignoring packages

gobyrne committed:
Revert "Ignoring packages"

This reverts commit 25b6f12418b555ad6d7216a8202f97ed1f2d1f50.

gobyrne committed:
Revert "Removing all those packages..."

This reverts commit b8c7ba483b8baff7d7ce89fcaee735fce7c89e9d.

gobyrne committed:
Reverting the bower.json deletion

gobyrne committed:
Ignoring app/bower_components/*

gobyrne committed:
Using jquery to submit all forms

kheaactua committed:
Merged in gitignore

kheaactua committed:
Whitespace and quotation marks

kheaactua committed:
Whitespace, modeline

kheaactua committed:
Fixed slash

kheaactua committed:
While jshints will yell at you for this, you don't need to declare it.
Because app.js is included first, `server` is still available.

Plus, typically all JS files are combined into one large file, so it's
"common" to have some global variables not be defined in one file.  With
angularjs it's typically the `app` variable.

kheaactua commented on app/index.html:
```diff
@@ -1,97 +1,102 @@
+				</li>
+			</ul>
+		</div>
+		<input class="btn btn-primary" type="submit" onclick="sendForms()">
```
This is outside of your controller/scope

kheaactua commented on app/script.js:
```diff
@@ -0,0 +1,16 @@
+var server = 'http://diplomacy3.asilika.com';
+
+sendForms = function(){
```
You gotta document what these do better

kheaactua committed:
Fixed left over

kheaactua committed:
Merge branch 'rest_api' into frontend

kheaactua committed:
Fixed up some bugs in Gray's code, removed the need for JQuery

kheaactua committed:
Doubly mapped neighbours.. Now they seem to export properly

kheaactua committed:
Put some comments explaning how the write\*Order function could be extended
